src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineControl_ModelComputation.cs

```diff
@@ -0,0 +1,257 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
nice.  appreciate this being pulled out into its own file to make it easier to understand the different responsibilities.